lang/es-es.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} no es válido.",
 	"components.form-element.defaultFieldLabel": "Campo",
 	"components.form-element.input.email.typeMismatch": "El correo electrónico no es válido",
-	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
-	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {El número debe ser inferior a {max}.} other {El número debe ser inferior o igual a {max}.}}",
-	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {El número debe ser superior a {min}.} other {El número debe ser superior o igual a {min}.}}",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {El número debe ser mayor que {min} y menor que {max}.} other {El número debe ser mayor que {min} y menor o igual que {max}.}}} other {{maxExclusive, select, true {El número debe ser mayor o igual que {min} y menor que {max}.} other {El número debe ser mayor o igual que {min} y menor o igual que {max}.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {El número debe ser menor que {max}.} other {El número debe ser menor o igual que {max}.}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {El número debe ser mayor que {min}.} other {El número debe ser mayor o igual que {min}.}}",
 	"components.form-element.input.text.tooShort": "{label} debe tener al menos {minlength} caracteres",
 	"components.form-element.input.url.typeMismatch": "La dirección URL no es válida",
 	"components.form-element.valueMissing": "{label} es obligatorio.",
-	"components.form-error-summary.errorSummary": "{count, plural, one {Se ha encontrado 1 error en la información enviada} other {Se han encontrado {count} errores en la información enviada}}",
+	"components.form-error-summary.errorSummary": "{count, plural, one {Se ha encontrado 1 error en la información enviada} other {Se han encontrado {count} errores en la información que ha enviado}}",
```

Comment:
Before `{count} errors found in submitted information` After `{count} errors were found in the information you submitted` After seems more consistent with the english term